### PR TITLE
[MM-11051] Add a loading_palceholder when user info is loading

### PR DESCRIPTION
**Pull request by sudheerDev**
#### Summary
Add a placehoder when user info is loading for LHS.

#### Ticket Link
[MM-11051](https://mattermost.atlassian.net/browse/MM-11051)

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Added or updated unit tests (required for all new features)
- [x] All new/modified APIs include changes to [mattermost-redux](https://github.com/mattermost/mattermost-redux) (please link)
- [x] Has UI changes

#### Screenshots
aug-07-2018 15-38-13

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,13 +15,14 @@
...
     "jsc-android": "216113.0.3",
-    "mattermost-redux": "github:mattermost/mattermost-redux#12353bc88f20dde53b6191747790c6a82b8dd527",
+    "mattermost-redux": "github:mattermost/mattermost-redux#f13022064f1ed24a0d1c085f433490f9d40cfc2f",
     "mime-db": "1.33.0",
     "prop-types": "15.6.1",
     "react": "16.3.2",
     "react-intl": "2.4.0",
     "react-native": "github:enahum/react-native#mm",
     "react-native-animatable": "1.2.4",
+    "react-native-animated-ellipsis": "sudheerDev/react-native-animated-ellipsis.git#326167b8e0fa3b6f0422c06be412bf177b725666",
```
[the remark this answers is not in the document]
Comment (enahum) [reply package.json]: Yeah not sure why this happened, have you tried different options?

[the remark this answers is not in the document]
Comment (sudheerDev) [reply package.json]: Tried. My package-lock changes to this even without adding a library. every time I do`npm install` lock file changes to the one in the commit

**Comment by sudheerDev** [comment]
@enahum I removed the changes to package.json. Can you please update them

**Comment by saturninoabril** [comment]
Until when we'll show this `LoadingPlaceholder`?  I asked because I noticed in the webapp (https://ci-linux-mysql.mattermost.com), the loading indicator never stops.  And also, do we really fetch the data somewhere when we show the loading indicator?

On the webapp:
loading

**Comment by sudheerDev** [comment]
@saturninoabril we make this call more than few times actually. 

https://github.com/mattermost/mattermost-webapp/blob/415bec8ce56dbc21f89615f7b5b6bd0397a2ce3d/actions/user_actions.jsx#L269.
This actually replicates the same as `someone` seen in app it just replacing with another component.

Most likely the user you are seeing is deleted for testing. To show a failed load of user we dont have a status. i.e if we ask for details of sidebar i.e say 20 users and if we do get only 19 then there is no status through which we can guess that a user failed to load. 

Ideally we should have status for not just API call but for induvidual entities as userID info is intiated and failed so that way we can stop the loading indicator and show probably `deleted user` text but i think it should be a story as it invoves good amount of changes on both web, mobile and redux.

**Comment by saturninoabril** [comment]
Yeah, it's probably a deleted user.  In case of cannot be resolved, I was thinking that it might be better not to show nothing at all rather than showing the loading indicator continuously.
Maybe it's something to keep an eye in the future.

**Comment by sudheerDev** [comment]
@saturninoabril We could do that but in that case users cannot remove `deleted users` from sidebar. They will just hide. I found a case where unreads from deleted users can make it crazy if we hide.

**Pull request merged**